l10n_cl_currency_rate_sbif/views/update_button.xml

```diff
@@ -0,0 +1,24 @@
+<!-- Copyright (C) 2019 Blanco Martín & Asociados
+     Copyright (C) 2019 Konos
+     Copyright (C) 2019 Open Source Integrators
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+
+    <record id="view_res_currency_form" model="ir.ui.view">
+        <field name="name">res.currency.sbif.form</field>
+        <field name="model">res.currency</field>
+        <field name="inherit_id" ref="base.view_currency_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="/form/sheet/div[@name='button_box']"
+                   position="inside">
+                <button string="Update Currency"
+                        type="object"
+                        name="action_update_currency"
+                        class="oe_stat_button"
+                        icon="fa-magnet"
+                        attrs="{'invisible':[('name', 'not in',
+                        ['SBIFUSD', 'SBIFUF', 'SBIFUTM', 'SBIFEUR'])]}"/>
```

Comment:
@max3903 `name` field takes 3 char only. It seems `Update Currency` button never visible. :thinking: